docker/Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV JAVA_HOME=/opt/java/openjdk
 ENV PATH "${JAVA_HOME}/bin:${PATH}"
 COPY --from=jre-build /javaruntime $JAVA_HOME
 
-RUN apt update && apt install -y --no-install-recommends curl iputils-ping net-tools netcat \
+RUN apt update && apt install -y --no-install-recommends libssl3 curl iputils-ping net-tools netcat \
```

Comment:
Are any of the other installs i.e. curl, iputils-ping, net-tools, netcat related to dependency upgrades?

Reply:
no ... they are/were used by DevOps in past. iirc the getting started scripts used these tools to determine if ethsigner is up or not.